dashboard/src/modules/components/OverviewComponent/NewRunsComponent.jsx

```diff
@@ -37,26 +37,31 @@ const NewRunsComponent = () => {
   const { newRuns, initNewRuns, selectedRuns } = useSelector(
     (state) => state.overview
   );
-  const loginDetails = useSelector((state) => state.userAuth.loginDetails);
+  // const loginDetails = useSelector((state) => state.userAuth.loginDetails);
```

Comment:
Can we just remove this code instead of leaving it commented out?

Reply:
It is required in the future.

Reply:
Commented out for future use.

Reply:
Given that the code used to be not-commented-out (and, given that we expect to have it not-commented-out again in the future), is there any reason why it needs to be commented out (in this particular spot) right now?

This code is just fetching a value from the Redact state, right?  It looks like the reducer for that part of the state has a value (albeit, it's an empty object, but it's a value!) for `loginDetails`.  So, it's not obvious to me how leaving this line uncommented will hurt anything (at least in the immediate-term future).

So, I suggest reverting this change.  And, the same applies to `dashboard/src/modules/components/OverviewComponent/SavedRunsComponent.jsx` line 35.

Reply:
To remove the unused variable warning, it is commented out

Reply:
It is good and appropriate to avoid the warning.  The lingering question is, is this piece of text so complicated, subtle, or erudite that we need to preserve it for future use, or is it reasonably obvious and straightforward enough that reconstructing it will pose little problem, and so it should just be removed?  That is, we should avoid having commented-out code in our files, and, so, unless there is a good reason to retain it, we should just delete it now and   then, once it is needed, return it.